api/src/main/resources/assets/nolijium/lang/zh_cn.json

```diff
@@ -41,18 +64,18 @@
     "nolijium.options.hud_shadow.name": "HUD 文本阴影",
     "nolijium.options.hud_shadow.tooltip": "如果启用，HUD 文本将带有阴影。",
     "nolijium.options.hud_refresh_rate_ticks.name": "HUD 刷新率",
-    "nolijium.options.hud_refresh_rate_ticks.tooltip": "HUD 刷新内容的频率。\"充分利用整个 CPU\" 选项将每帧刷新一次。较低的值可能会降低性能。",
+    "nolijium.options.hud_refresh_rate_ticks.tooltip": "HUD 刷新内容的频率。 §l\"充分利用整个 CPU\"§r 选项将每帧刷新一次。较低的值可能会降低性能。",
```

Comment:
Were the quotes kept intentionally? I removed them from the `en_us` translation. If they were kept intentionally for readability in the `zh_cn` locale that's fine, just wanted to check.